examples/ahiqar-arabic-karshuni-local-de.html

```diff
@@ -0,0 +1,159 @@
+<!DOCTYPE html>
```

Comment:
Do we still need this?

Reply:
no, we don't actually. We use glf-local file to test the german translations. Thus, I removed it